docs/send-data/sumo-logic-distribution-for-opentelemetry-collector/installation.md

```diff
@@ -0,0 +1,405 @@
+
+#### Linux on amd64 (x86-64)
+
+1. Download the release binary:
+    ```bash
+    curl -sLo otelcol-sumo "https://github.com/SumoLogic/sumologic-otel-collector/releases/download/v0.57.2-sumo-1/otelcol-sumo-0.57.2-sumo-1-linux_amd64"
+    ```
+1. Install the release binary in your `PATH`:
+    ```bash
+    chmod +x otelcol-sumo
+    sudo mv otelcol-sumo /usr/local/bin/otelcol-sumo
+    ```
+1. [Verify the installation](#verify-the-installation)
+1. Prepare the configuration according to [this](/docs/send-data/sumo-logic-distribution-for-opentelemetry-collector/configuration) document and save it in `config.yaml`.
+   :::note
+   It is recommended to limit access to the configuration file as it contains sensitive information. You can change access permissions to the configuration file using:
+   ```bash
+   chmod 640 config.yaml
+   ```
+   :::
+1. Run Sumo Logic Distribution for OpenTelemetry Collector:
+   ```bash
+   otelcol-sumo --config config.yaml
+   ```
+
+#### Linux on arm64
+
+1. Download the release binary:
+    ```bash
+    curl -sLo otelcol-sumo "https://github.com/SumoLogic/sumologic-otel-collector/releases/download/v0.57.2-sumo-1/otelcol-sumo-0.57.2-sumo-1-linux_arm64"
+    ```
+1. Install the release binary in your `PATH`:
+    ```bash
+    chmod +x otelcol-sumo
+    sudo mv otelcol-sumo /usr/local/bin/otelcol-sumo
+    ```
+1. [Verify the installation](#verify-the-installation)
+1. Prepare the configuration according to [this](/docs/send-data/sumo-logic-distribution-for-opentelemetry-collector/configuration) document and save it in `config.yaml`.
+   :::note
+   It is recommended to limit access to the configuration file as it contains sensitive information. You can change access permissions to the configuration file using:
+   ```bash
+   chmod 640 config.yaml
+   ```
+   :::
+1. Run Sumo Logic Distribution for OpenTelemetry Collector:
+   ```bash
+   otelcol-sumo --config config.yaml
+   ```
+
+#### MacOS on amd64 (x86-64)
+
+1. Download the release binary:
+    ```bash
+    curl -sLo otelcol-sumo "https://github.com/SumoLogic/sumologic-otel-collector/releases/download/v0.57.2-sumo-1/otelcol-sumo-0.57.2-sumo-1-darwin_amd64"
+    ```
+1. Install the release binary in your `PATH`:
+    ```bash
+    chmod +x otelcol-sumo
+    sudo mv otelcol-sumo /usr/local/bin/otelcol-sumo
+    ```
+1. [Verify the installation](#verify-the-installation)
+1. Prepare the configuration according to [this](/docs/send-data/sumo-logic-distribution-for-opentelemetry-collector/configuration) document and save it in `config.yaml`.
+   :::note
+   It is recommended to limit access to the configuration file as it contains sensitive information. You can change access permissions to the configuration file using:
+   ```bash
+   chmod 640 config.yaml
+   ```
+   :::
+1. Run Sumo Logic Distribution for OpenTelemetry Collector:
+   ```bash
+   otelcol-sumo --config config.yaml
+   ```
+
+#### MacOS on arm64 (Apple M1) (x86-64)
```

Comment:
This is incorrect (also in source docs at https://github.com/SumoLogic/sumologic-otel-collector/blob/v0.71.0-sumo-0/docs/installation.md#macos-on-arm64-apple-m1-x86-64).
```suggestion
#### MacOS on arm64 (Apple M1)
```